WalletLibrary/WalletLibrary/Networking/Operations/OpenID4VCIRequestNetworkOperation.swift

```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+/**
+ * The Network Operation to prefer the OpenID4VCI Request.
+ */
+struct OpenID4VCIRequestNetworkOperation: WalletLibraryFetchOperation
+{
+    typealias ResponseBody = Data
+    typealias Decoder = ServiceResponseDecoder
+    
+    var decoder = ServiceResponseDecoder()
+    
+    var urlSession: URLSession
+    
+    var urlRequest: URLRequest
+    
+    var correlationVector: VerifiedIdCorrelationHeader?
+    
+    let PreferHeaderValue = "oid4vci-interop-profile-version=0.0.1"
```

Comment:
nit: this prefer header value is specific to oid4vci interop. Could you rename to reflect that?

Reply:
will do!